pkg/crypto/measure.go

```diff
@@ -30,7 +30,12 @@ func TryMeasureBootConfig(name, kernel, initramfs, kernelArgs, deviceTree string
 	TryMeasureData(BootConfig, []byte(initramfs), initramfs)
```

Comment:
note: file names should not be measured, or at least should offer a sane default to skip name measurement. The risk with name measurements is that this becomes an operational pain with no real benefit, since the actual content is already verified. This can be done in another PR if you prefer

Reply:
done